README.md

```diff
@@ -72,17 +72,17 @@ CATALINA_OPTS="(here we set settings for JMX monitoring)"
 sudo cp ~/openrouteservice/openroutesrvice/target/openroutservice-4.7.0.war /var/lib/tomcat8/webapps/ors.war
 ```
 
-  5. Tomcat should now automatically detect the new WAR file and deploy the service. Depending on profiles and size of the OSM data, this can take some time until openrouteservice has built graphs and is ready for generating routes. You can check if it is ready by accessing `http://localhost:8080/ors/health` (the port and URL may be different if you have installed Tomcat differently than above). If you get a `status: ready` message, you are good to go in creating routes.
+  5. Tomcat should now automatically detect the new WAR file and deploy the service. Depending on profiles and size of the OSM data, this can take some time until openrouteservice has built graphs and is ready for generating routes. You can check if it is ready by accessing `http://localhost:8082/ors/health` (the port and URL may be different if you have installed Tomcat differently than above). If you get a `status: ready` message, you are good to go in creating routes.
 
 There are numerous settings within the `app.config` which are highly dependent on your individual circumstances, but many of these are documented. As a guide however you can look at the `app.config.sample` file in the `docker/conf` folder. If you run into issues relating to out of memory or similar, then you will need to adjust java/tomcat settings accordingly.
 
 ## Usage
 
 Openrouteservice offers a set of endpoints for different spatial purposes. They are served with the help of [Tomcat in a java servlet container](https://github.com/GIScience/openrouteservice/blob/master/openrouteservice/WebContent/WEB-INF/web.xml). By default you will be able to query the services with these addresses:
 
-- `http://localhost:8080/name_of_war_archive/routes`
-- `http://localhost:8080/name_of_war_archive/isochrones`
-- `http://localhost:8080/name_of_war_archive/matrix`
+- `http://localhost:8082/v2/routes`

```

Comment:
Again, if running through Tomcat on the machine, the port of 8080 is correct, and you need the name of the war file (ors). Normally it is better to also use "directions" rather than "routes" as routes is just there to maintain a bit of backwards compatibility.

So a directions request should normally be `http://localhost:8080/ors/v2/directions`